src/test/pybind/test_rados.py

```diff
@@ -528,6 +528,13 @@ def test_write_ops(self):
             with assert_raises(ObjectNotFound):
                 self.ioctx.read('write_ops')
 
+    def test_execute_op(self):
+        write_op = self.ioctx.create_write_op()
+        write_op.write_op_exec("hello", "record_hello", "ebs")
+        comp = self.ioctx.operate_write_op(write_op, "object")
+        self.ioctx.release_write_op(write_op)
+        eq(self.ioctx.read('object'), b"Hello, ebs!")
```

Comment:
could you use `WriteOpCtx` instead? like
```py
with WriteOpCtx() as write_op:
  write_op.exec("hello", "record_hello", "ebs")
  self.ioctx.operate_write_op(write_op, "object")
eq(self.ioctx.read('object'), b"Hello, ebs!")
```
 also, could you remind me where `write_op_exec()` is defined?